mlshvch/hometask-5/2.sql

```diff
@@ -0,0 +1,5 @@
+SELECT f.title, l.name
+FROM film f
+         LEFT JOIN language l
+                   ON f.language_id = l.language_id
```

Comment:
That's purely subjective, but I don't see the issue with JOIN clause and ON being on the same line
```suggestion
LEFT JOIN language l ON f.language_id = l.language_id
```